src/main/java/io/reactivex/schedulers/Schedulers.java

```diff
@@ -43,56 +45,105 @@
         NEW_THREAD = RxJavaPlugins.initNewThreadScheduler(NewThreadScheduler.instance());
     }
     
-    /*
-     * TODO I started to like enums for singletons and non-instantiatable
-     * utility classes, but since this is part of the public API,
-     * that would act quite unorthodoxically.
-     */
     private Schedulers() {
         throw new IllegalStateException("No instances");
     }
     
+    /**
+     * Creates and returns a {@link Scheduler} intended for computational work.
+     * <p>
+     * This can be used for event-loops, processing callbacks and other computational work.
+     * <p>
+     * Do not perform IO-bound work on this scheduler. Use {@link #io()} instead.
+     * <p>
+     * Unhandled errors will be delivered to the scheduler Thread's {@link java.lang.Thread.UncaughtExceptionHandler}.
+     *
+     * @return a {@link Scheduler} meant for computation-bound work
+     */
     public static Scheduler computation() {
         return RxJavaPlugins.onComputationScheduler(COMPUTATION);
     }
     
+    /**
+     * Creates and returns a {@link Scheduler} intended for IO-bound work.
+     * <p>
+     * The implementation is backed by an {@link Executor} thread-pool that will grow as needed.
+     * <p>
+     * This can be used for asynchronously performing blocking IO.
+     * <p>
+     * Do not perform computational work on this scheduler. Use {@link #computation()} instead.
+     * <p>
+     * Unhandled errors will be delivered to the scheduler Thread's {@link java.lang.Thread.UncaughtExceptionHandler}.
+     *
+     * @return a {@link Scheduler} meant for IO-bound work
+     */
     public static Scheduler io() {
         return RxJavaPlugins.onIoScheduler(IO);
     }
     
-    public static TestScheduler test() {
+    /**
+     * Creates and returns a {@code TestScheduler}, which is useful for debugging. It allows you to test
+     * schedules of events by manually advancing the clock at whatever pace you choose.
+     *
+     * @return a {@code TestScheduler} meant for debugging
+     */
+    public static TestScheduler test() { // NOPMD
         return new TestScheduler();
```

Comment:
Traditional how? That it's in 1.x? I don't see why that should stop us from removing it.

Doing something like this is wrong:
```java
timer(1, SECONDS, Schedulers.test())
```
yet it's how you'd use every other static method in this class.